docs/modules/policies/pages/index.adoc

```diff
@@ -10,7 +10,7 @@ There are five kinds of Cerbos policies:
 xref:derived_roles.adoc[Derived roles]:: Traditional RBAC roles are usually broad groupings with no context awareness. Derived roles are a way of augmenting those broad roles with contextual data to provide more fine-grained control at runtime. For example, a person with the broad `manager` role can be augmented to `manager_of_scranton_branch` by taking into account the geographic location (or another factor) and giving that derived role bearer extra privileges on resources that belong to the Scranton branch.
 xref:resource_policies.adoc[Resource policies]:: Defines rules for actions that can be performed on a given resource. A resource is an application-specific concept that applies to anything that requires access rules. For example, in an HR application, a resource can be as coarse-grained as a full employee record or as fine-grained as a single field in the record.
 xref:principal_policies.adoc[Principal policies]:: Defines overrides for a specific user.
-xref:role_policies.adoc[Role policies]:: Define rules specific to a given role. Rules are defined as a list of permissible actions that apply to a particular resource. Role policies are evaluated before resource policies but don't guarantee an `ALLOW` in the case of a matching rule -- resource policies must also resolve to an `ALLOW` for the same request. Follow the link for a more detailed description.
+xref:role_policies.adoc[Role policies]:: Define rules specific to a given role. Rules are defined as a list of permissible actions that apply to a particular resource. Role policies are evaluated before resource policies but don't guarantee an `ALLOW` in the case of a matching rule -- resource policies must also resolve to an `ALLOW` for the same request. Follow xref:role_policies.adoc[the link] for a more detailed description.
```

Comment:
"Click here" type of links were frowned upon by the professional technical writers that I used to work with. So, I am just blindly propagating that advice here 😄 

```suggestion
xref:role_policies.adoc[Role policies]:: Define rules specific to a given role. Rules are defined as a list of permissible actions that apply to a particular resource. Role policies are evaluated before resource policies but don't guarantee an `ALLOW` in the case of a matching rule -- resource policies must also resolve to an `ALLOW` for the same request. See the dedicated xref:role_policies.adoc[role policy documentation] for more details.
```

Reply:
Oh whoops, that's good to know!